vmdb/spec/models/ems_refresh/refreshers/foreman_refresher_spec.rb

```diff
@@ -134,6 +131,29 @@ def assert_configured_system
     expect(system.configuration_profile).to   eq(child)
   end
 
+  it "loads data with locations" do
```

Comment:
@Fryguy I think we need both cases since locations and organizations are all or nothing.  We can't have some records with and some without.

Maybe we should break the EMS Refresh specs out of VMDB?